src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,7 +1,13 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.NotNull;
+import mate.academy.spring.validation.Email;
+
 public class UserRequestDto {
+    @NotNull
+    @Email
```

Comment:
```suggestion
    @Email
```
lets combine them, and move check for null to validator